docker-images/pysyft-worker/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Better to not run the worker server without ID
+if [ -z "${WORKER_ID}" ]; then
+  echo "[-] You need to set the WORKER_ID environment variable"
+  exit
+fi
+
+python worker-server.py --id $WORKER_ID --port 8777
```

Comment:
Use `exec python ...` to pass python PID to docker, not shell PID.
Otherwise docker will not be able to terminate the process normally.

Reply:
done